lint-file.js

```diff
@@ -0,0 +1,17 @@
+const path =require('path');
+
+const kebabCase = /^[a-z0-9]*(?:[a-z0-9+-])+[a-z0-9]+$/;
```

Comment:
maybe instead coding regular expressions manually, use packages like [`camelcase`](https://www.npmjs.com/package/camelcase) and [`decamelize`](https://www.npmjs.com/package/decamelize) and test filenames like this 

```js
const isValid = decamelize(basename) === basename;
```

You'll have different styles supported almost for free.

Reply:
done